package.json

```diff
@@ -102,6 +103,22 @@
     "live-tests-ws-php": "npm run live-tests -- --php    --ws",
     "live-tests-rest-csharp": "npm run live-tests -- --csharp",
     "live-tests-ws-csharp": "npm run live-tests -- --csharp --ws",
+    "test-request-ts":         "npm run runtests -- --request --ts",
+    "test-request-js":         "npm run runtests -- --request --js",
+    "test-request-py-sync":    "npm run runtests -- --request --python",
+    "test-request-py-async":   "npm run runtests -- --request --python-async",
+    "test-request-php-sync":   "npm run runtests -- --request --php",
+    "test-request-php-async":  "npm run runtests -- --request --php-async",
+    "test-request-cs":         "npm run runtests -- --request --csharp",
+    "test-request":            "npm run runtests -- --request",
+    "test-response-ts":        "npm run runtests -- --response --ts",
+    "test-response-js":        "npm run runtests -- --response --js",
+    "test-response-py-sync":   "npm run runtests -- --response --python",
+    "test-response-py-async":  "npm run runtests -- --response --python-async",
+    "test-response-php-sync":  "npm run runtests -- --response --php",
+    "test-response-php-async": "npm run runtests -- --response --php-async",
+    "test-response-cs":        "npm run runtests -- --response --csharp",
+    "test-response":           "npm run runtests -- --response",
```

Comment:
we have been doing all other test-related operations through run-tests, which provides uniform output (colored FAILED, SUCCESS, WARNING, with `[INFO]` messages formatted etc..), so we should try to have a support of running any kind of tests through run-tests. at this stage, only adding commands into package.json  (not migrating build.sh commands to them for a while).

Reply:
@ttodua this is the case where I don't think this is needed, why add an extra layer of complexity? static tests already print out the proper success/error message

Reply:
How do you run request or response tests for all exchanges parallelly?

Reply:
done